LICENSE.md

```diff
@@ -67,58 +67,59 @@ The User may not limit access to the Mod in any way, except as provided by the G
 The User may decompile compiled binaries of the Mod.  
 The User may examine the Mod's source code.
 
-5. Right to distribute
+5. No right to distribute
 --------
 
-The Author reserves the right to distribute the Mod.  
-The Author may designate a person or organization as a Distributor.  
-A Distributor is granted the right to distribute the Mod.  
-The Author may require that a Distributor provides a direct link to the Codebase.  
-The User may not distribute the Mod without being granted distribution rights from the Author.  
-The Author may revoke distribution rights from any Distributor.  
+The Author reserves the right to transmit verbatim copies of ("Distribute") the Mod.  
+Rights to distribute the Mod must be granted in a separate license which must be explicitly obtained from the Author.
```

Comment:
I can't speak for other hosting providers, but Dreamhost doesn't have that. You can only use their services to distribute stuff that you have a right to distribute, but there's nothing granting that right to Dreamhost.